cluster/kind-replica1.sh

```diff
@@ -27,9 +29,21 @@ kubeadmConfigPatches:
   topologyManagerPolicy: "single-numa-node"
   reservedSystemCPUs: "1"
 nodes:
-- role: control-plane
-- role: worker
 EOF
+counter=1
+while [ $counter -le $MASTER_NUM ]
+do
+ echo "- role: control-plane" >> ${CONTEXT}/kindconfig.yaml
+((counter++))
+done
+
```

Comment:
I prefer this idion:

```bash
for idx in $( seq 1 $MASTER_NUM ); do
echo "- role: control-plane" >> ${CONTEXT}/kindconfig.yaml
done
```

but yours works too so let's go ahead

Reply:
Yeah your suggestion looks much neater! If you want to revert the merge, I can update it as part of this PR itself or submit as another PR? I'll let you decide.

Reply:
no worries, we can improve later. Let's move forward. And thanks for your PRs!

Reply:
Ack, Thanks!